doc/did-method-spec.md

```diff
@@ -237,11 +259,26 @@ The only 2 `delegateTypes` that are currently published in the DID document are:
   corresponding entry to the `authentication` section.
 
 Note, the `delegateType` is a `bytes32` type for Ethereum gas efficiency reasons and not a `string`. This restricts us
-to 32 bytes, which is why we use the short hand versions above.
+to 32 bytes, which is why we use the short-hand versions above.
 
-Only events with a `validTo` in seconds greater or equal to the current time should be included in the DID document.
+Only events with a `validTo` (measured in seconds) greater or equal to the current time should be included in the DID
+document. When resolving an older version (using `versionId` in the didURL query string), the `validTo` entry MUST be
+compared to the timestamp of the block of `versionId` height.
 
-#### Non-Ethereum Attributes
+Such valid delegates MUST be added to the `verificationMethod` array as `EcdsaSecp256k1RecoveryMethod2020` entries, with
+`delegate` listed under `blockchainAccountId` and suffixed with `@eip155:<chainId>`
```

Comment:
should we define the `"-1"` part as well (in the delegate DID URI)?

Reply:
good point. This deserves to be its own section about the stability of key IDs

Reply:
I added a key ID section